docs/UserGuide.md

```diff
@@ -211,14 +211,14 @@ Multiple headers/tags must be unique and duplicates will be ignored
 
 <div markdown="span" class="alert alert-warning">:bulb: **Tip:**
 When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative <br>
```

Comment:
`person` needs to be replaced with `endpoint` here (occurring at a few other places as well).

Reply:
great catch